.devcontainer/docker-compose.yml

```diff
@@ -19,7 +19,7 @@ services:
     # (Adding the "ports" property to this file will not forward from a Codespace.)
 
   db:
-    image: postgres:latest
+    image: postgres:14-alpine
```

Comment:
it is a good practice to set a specific version of docker images to keep compatibility and make sure all students have the same dev environment